core/src/main/java/com/linecorp/armeria/common/Cookie.java

```diff
@@ -68,11 +68,25 @@ static Cookie of(String name, String value) {
      *
      * @param name the name of the {@link Cookie}
      * @param value the value of the {@link Cookie}
+     *
+     * @deprecated Use {@link #secureBuilder(String, String)} instead to create secure cookie.
      */
+    @Deprecated
     static CookieBuilder builder(String name, String value) {
```

Comment:
Q) Is it possible to migraate usages of `builder` to `secureBuilder()`?

Armeria users may refer to examples, tests to see how `CookieBuilder` is used. I think it would be good to promote usage of `secureBuilder` through our codebase.

(For internal usages, I think it is fine to keep using the deprecated `builder()`. Let me know if you feel otherwise other maintainers cc. @minwoox @ikhoon @trustin )

Reply:
That's true. Let me check the usage of the current `builder` then replace them if possible.